### ISSUE TITLE
Errors on "import hypergan"

### ISSUE THREAD

**Issue by Quasimondo** [255BITS/HyperGAN#30]
It is probably something I did wrong, but I thought I let you know.

I just installed HyperGAN from source using python3 setup.py develop

The first error I got when trying "import hypergan" in a Python console was that the HyperChamber module was missing. After installing that one separated this error went away.

But now I am stuck with this. It seems like the hypergan.regularizers classes do not get installed properly (at least over here):

> >>> import hypergan
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "hypergan/__init__.py", line 2, in <module>
>     from .gan import GAN
>   File "hypergan/gan.py", line 13, in <module>
>     from hypergan.discriminators import *
>   File "hypergan/discriminators/densenet_discriminator.py", line 5, in <module>
>     import hypergan.regularizers.minibatch_regularizer as minibatch_regularizer
> ImportError: No module named regularizers.minibatch_regularizer

Any ideas?

**Comment by Quasimondo** [comment]
Ah - I should mention that since the python3 installation failed entirely for me I tried this with python 2.7 - which is probable the cause for these errors.

**Comment by Quasimondo** [comment]
Okay, sorry for the noise, but just to close this up and in case it is helpful for someone:

Looks like my python3 installation is messed up, nevertheless I could eventually successfully install HyperGAN using 
`sudo python3 setup.py develop`

After that I still had to install HyperChamber (same way using 'sudo python3 setup.py develop' in the HyperChamber folder - so not 'python setup.py develop' as the readme says). I also had to pip3 install flask and pip3 install flask_cors. Not sure if all this is handled if I had used the pip installer for HyperGAN, but of course I wanted the latest version.
